drivers/mtd/Kconfig

```diff
@@ -5,6 +5,88 @@
 # directory for more details.
 #
 
-config MODULE_MTD
+config HAVE_MTD_AT24CXXX
+    bool
+    depends on MODULE_AT24CXXX
+    select MODULE_MTD_AT24CXXX if MODULE_MTD
+    help
+        Indicates that a at24cxxx EEPROM MTD is present
+
+config HAVE_MTD_AT25XXX
+    bool
+    select MODULE_MTD_AT25XXX if MODULE_MTD
```

Comment:
Shouldn't this also depend on `MODULE_AT25XXX`? And I guess the driver selects this feature right?